src/app/static/src/app/components/uploadInputs/UploadInputs.vue

```diff
@@ -146,45 +146,45 @@
                     error: state.pjnzError,
                     fromADR: !!state.pjnz?.fromADR,
                     existingFileName: (state.pjnz && state.pjnz.filename) || state.pjnzErroredFile
-                } as PartialFileUploadProps),
```

Comment:
Do you have to remove this line ?

Reply:
we don't really gain anything from this, the properties are recognised as the correct types by typescript